governance/votes/Trial Executive vote - July 12, 2023.md

```diff
@@ -0,0 +1,191 @@
+- The FlapperUniV2 (Smart Burn Engine) will be added to replace Surplus Auctions (flaps).
+- Multiple Cron Jobs will be added to Chainlog
+- A new Multiswap Conduit will be added for RWA015-A.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. 
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+
+---
+
+## Proposal Details
+
+### Stability Scope Defined Parameter Changes
+
+As per the [Stability Scope](https://mips.makerdao.com/mips/details/MIP104), and the [adjustments to the base rate](https://forum.makerdao.com/t/stability-scope-parameter-changes-3/21238/6), the following parameter changes will be made, if this executive proposal passes:
+
+**[Dai Savings Rate (DSR)](https://manual.makerdao.com/parameter-index/core/param-dai-savings-rate) Decrease**
+- Decrease the DSR by 0.30% from 3.49% to 3.19%
+
+**[Liquidation Ratio](https://manual.makerdao.com/parameter-index/vault-risk/param-liquidation-ratio) Changes**
+- Reduce WSTETH-A Liquidation Ratio by 10% from 160% to 150%
+- Reduce WSTETH-B Liquidation Ratio by 10% from 185% to 175%
+- Reduce RETH-A Liquidation Ratio by 20% from 170% to 150%
+
+**[Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) Reductions**
+- Reduce the ETH-A Stability Fee (SF) by 0.30% from 3.74% to 3.44%
+- Reduce the ETH-B Stability Fee (SF) by 0.30% from 4.24% to 3.94%
+- Reduce the ETH-C Stability Fee (SF) by 0.30% from 3.49% to 3.19%
+- Reduce the WSTETH-A Stability Fee (SF) by 0.30% from 3.74% to 3.44%
+- Reduce the WSTETH-B Stability Fee (SF) by 0.30% from 3.49% to 3.19%
+- Reduce the RETH-A Stability Fee (SF) by 0.30% from 3.74% to 3.44%
+- Reduce the WBTC-A Stability Fee (SF) by 0.11% from 5.80% to 5.69%
+- Reduce the WBTC-B Stability Fee (SF) by 0.11% from 6.30% to 6.19%
+- Reduce the WBTC-C Stability Fee (SF) by 0.11% from 5.55% to 5.44%
+
+#### CRVV1ETHSTETH-A 1st Stage Offboarding
+
+As per [MIP104 ](https://mips.makerdao.com/mips/details/MIP104#0-the-stability-scope) the debt ceiling (``line``) for ``CRVV1ETHSTETH-A`` will be set to ``0``. 
+
+This change will be happening alongside the other [Stability Scope Parameter changes](https://forum.makerdao.com/t/stability-scope-parameter-changes-3/21238/6).
```

Comment:
This line just invites more questions than answers. What other changes? Why call this out as happening 'alongside' the whole executive happens in one block.... WHAT DOES IT MEAN?!?!